lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1573,6 +1573,7 @@ def _resource_sku_to_capability(  # noqa: C901
                 schema.FeatureSettings.create(features.StartStop.name()),
                 schema.FeatureSettings.create(features.SerialConsole.name()),
                 schema.FeatureSettings.create(features.Hibernation.name()),
+                schema.FeatureSettings.create(features.Resize.name()),
```

Comment:
My understanding is not all vm size can be resize. Is there a list of it?

Reply:
probably no, but when resize a vm, I know there is an api to know which vm size can be resized based on current vm size. if no sizes listed, then can skip the case, I will check.